charts/collector/values.yaml

```
@@ -32,7 +32,7 @@ collector:
   service:
     type: ClusterIP
     port: 5000
-  # Set this to the number of requests that will remain in the PostgreSQL post daily cleaning
+  # Set this to the number of allowed consecutive minutes of abnormal traffic, meaning that there are sudden unexpected traffic drops or increases, or the traffic pattern is suddenly erratic. If traffic remains abnormal past the set time limit, it triggers an alert.
```

```suggestion
 # Set this to the maximum number of requests to keep in the PostgreSQL database. Any requests over this number are cleaned out daily, starting from the oldest request.
```
Bit of a mixup, this definition should be below.